server/datasources.json

```diff
@@ -1,6 +1,6 @@
 {
   "recipeDS": {
-    "url": "mongodb://heroku_p0dxgncb:gl2rr2bi235aoqfdojelqspjn1@ds147052.mlab.com:47052/heroku_p0dxgncb",
```

Comment:
I added this file by mistake.

Reply:
no prob. later i'll explain how we trying to update logic for basic static things like db link, api keys, etc.
it's quite hard sometimes to figure out what database belong to.

Reply:
here we'll address to this improvement: https://github.com/GroceriStar/groceristar/issues/518